lumigator/python/mzai/backend/backend/records/jobs.py

```diff
@@ -0,0 +1,19 @@
+import uuid
+from typing import Any
+
+from sqlalchemy import ForeignKey
+from sqlalchemy.orm import Mapped, mapped_column
+
+from backend.records.base import BaseRecord
+from backend.records.mixins import DateTimeMixin, JobStatusMixin, NameDescriptionMixin
+
+
+class JobRecord(BaseRecord, NameDescriptionMixin, JobStatusMixin, DateTimeMixin):
+    __tablename__ = "jobs"
+
+
+class JobResultRecord(BaseRecord):
+    __tablename__ = "job-results"
+
+    job_id: Mapped[uuid.UUID] = mapped_column(ForeignKey("jobs.id"), unique=True)
```

Comment:
Note that here we now set the job_id as the UUID rather than at the experiment level